packages/data/src/redux-store/metadata/selectors.js

```diff
@@ -140,7 +140,14 @@ export function getCachedResolvers( state ) {
  * @return {boolean} True if one or more selectors are resolving, false otherwise.
  */
 export function hasResolvingSelectors( state ) {
-	return [ ...Object.values( state ) ].some( ( selectorState ) =>
+	return Object.values( state ).some( ( selectorState ) =>
+		/**
+		 * This uses the internal `_map` property of `EquivalentKeyMap` for
+		 * optimization purposes, since the `EquivalentKeyMap` implementation
+		 * does not support a `.some()` implementation.
```

Comment:
```suggestion
		 * This uses the internal `_map` property of `EquivalentKeyMap` for
		 * optimization purposes, since the `EquivalentKeyMap` implementation
		 * does not support a `.values()` implementation.
```

Unless I misunderstood, I believe you have a typo here. By analogy with the standard `Map`, we expect the prototype to contain `values()` but not `some()` (besides, what would this `some()` iterate over? Just the values? Keys? Keys and values?).

Reply:
Oh well, `.some()` is what we intend to use here and it is what we actually use from `Array.prototype` once we convert the map to an array. We could rephrase this to use `.values()` or `.entries()` but I think since `.some()` is what we are practicaly using, it makes most sense to mention that in the documentation. We could mention `.values()` but it wouldn't immediately be clear as to where we'd need `.values()` since we never call `Map.prototype.values()`. Does this make sense?

Reply:
Hmm, I actually think I misunderstood you. Let me correct this.

Reply:
Updated. Thanks for the feedback!